src/components/[guild]/JoinModal/utils/processConnectorError.ts

```diff
@@ -8,9 +10,25 @@ const processConnectorError = (error: string): string | undefined => {
     return undefined
 
   try {
-    const matchedError = error.replaceAll("\\", "").match(/{"msg":"(.*)"}/gm)
-    const parsedError = JSON.parse(matchedError?.[0])
-    return capitalize(parsedError?.msg ?? "")
+    const matchedPlatformId = error.match(/^"\d" /g)
+    const platformName = matchedPlatformId?.length
+      ? platforms[
+          PlatformType[parseInt(matchedPlatformId[0].replace('"', "").trim())]
+        ].name
+      : null
+    const cleanError = error.replaceAll("\\", "")
+    const matchedMsg = cleanError.match(/{"msg":"(.*)"}/gm)
+    const matchedError = cleanError.match(/error: "(.*)"/gm)
+
+    const parsedError = JSON.parse(
+      matchedMsg?.[0] ??
+        (matchedError?.[0]
```

Comment:
Converting the error message to the `{ msg: "..." }` format, so we can use it easily inside `capitalize()`

Reply:
I'm not sure if it makes sense here or not, but without the `g` flag we could just access the captured group, and do `matchedMsg ?? matchedError` I think, what do you think?

I mean something like this:
```ts
const matchedMsg = cleanError.match(/{"msg":"(.*?)"}/m)?.[1]
```

Reply:
Oh, really nice catch, thanks, updated it in 2d5c527bb2ff1ca85e0a2e314a7cfaea3e8fe150
The `.match()` method actually always returns an array, but I removed the `g` flag and desctructured the array, so the code is a bit cleaner now.